### PR TITLE
Describe installation with Symfony Flex

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,46 +38,25 @@ After creating an Order via Admin panel, this new Order is listed like any other
...
+3. Copy plugin migrations to your migrations directory (e.g. `src/Migrations`) and apply them to your database:
 
     ```bash
-    bin/console assets:install --symlink web
+    cp -R vendor/sylius/admin-order-creation-plugin/migrations/* src/Migrations
+    bin/console doctrine:migrations:migrate
     ```
 
-8. Clear cache:
+#### Beware!
```
Comment (lchrusciel) [inline README.md]: I think we should put it just below Installation header

File: docs/legacy_installation.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+### Legacy installation (without Symfony Flex)
+
+1. Require plugin with composer:
+
+    ```bash
+    composer require sylius/admin-order-creation-plugin
+    ```
+
+2. Import configuration:
```
Comment (pamil) [inline docs/legacy_installation.md]: Could we provide which file should be modified? `app/config/config.yml` it this case,

File: docs/legacy_installation.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+    ```yaml
+    imports:
+        - { resource: "@SyliusAdminOrderCreationPlugin/Resources/config/app/config.yml" }
+    ```
+
+3. Import routing:
```
Comment (pamil) [inline docs/legacy_installation.md]: Same here, `app/config/routing.yml`.

File: docs/legacy_installation.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+4. Add plugin class to your `AppKernel`:
+
+    ```php
+    $bundles = [
+        new \FOS\JsRoutingBundle\FOSJsRoutingBundle(),
+        new \Sylius\AdminOrderCreationPlugin\SyliusAdminOrderCreationPlugin(),
+    ];
+    ```
+
+5. Copy Sylius templates overridden in plugin to your templates directory (e.g `templates/bundles/`):
```
Comment (pamil) [inline docs/legacy_installation.md]: Pre-Flex installations do not have the required config for `templates/` directory, they should use the old `app/Resources/somethinghereidontrememberanymore`.

**Pull request merged**